README.md

```diff
@@ -1,6 +1,8 @@
 # Chabok API for PHP
 Integrate your app with Chabok messaging & push api seamlessly
 
+**Important Note: Change all `Swagger` prefix to `Chabok`**
```

Comment:
Use a separate line for title formatting,
retitle to `[MUST READ] Upgrate Notes:`
use bullets for notes list